app/Gio.java

```diff
@@ -65,4 +66,19 @@ static void wakeupMainThread() {
 	}
 
 	static private native void scheduleMainFuncs();
+
+	static Intent startForegroundService(Context ctx, String title, String text) {
+		Intent intent = new Intent();
+		intent.setClass(ctx, GioForegroundService.class);
+		try {
+			intent.setClass(ctx, ctx.getClassLoader().loadClass("org/gioui/GioForegroundService"));
+		} catch (ClassNotFoundException e) {
```

Comment:
You don't need to coerce ClassNotFoundException to RuntimeException. It's simpler to just declare the method can throw it.